ding/entry/random_collect.py

```diff
@@ -0,0 +1,21 @@
+from ding.policy import PolicyFactory
+
+def random_collect_fn(policy_cfg, policy, collector, collector_env, commander, replay_buffer, mark_not_expert=False):
+    assert policy_cfg.random_collect_size > 0
```

Comment:
add unittest for this function, both episode and sample collector, both transition_with_policy_data True or False

Reply:
Resolved. Also add whether to use postprocess data function